releasenotes/notes/30294.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+- 30295
+releaseNotes:
+- |
+  **Added** OIDC JWT authenticator that supports both JWKS-URI and OIDC discovery. The OIDC JWT authenticator can be configured through the JWT_RULE env variable and when configured, it will be used as an istiod authenticator.
```

Comment:
comma in the last sentence isn't correct.  Maybe `The OIDC JWT authenticator will be used when configured through the JWT_RULE env variable.

Reply:
The PR has been revised according to the comment.